src/applications/common/schemaform/save-in-progress/FormStartControls.jsx

```diff
@@ -18,22 +18,29 @@ class FormStartControls extends React.Component {
     }
   }
 
+  goToIntroduction = () => {
+    this.props.router.push('/introduction');
+  }
+
   goToBeginning = () => {
     this.props.router.push(this.props.startPage);
   }
 
   handleLoadPrefill = () => {
+    const { beforeStartForm } = this.props.form;
     if (this.props.prefillAvailable) {
-      this.props.fetchInProgressForm(this.props.formId, this.props.migrations, true, this.props.prefillTransformer);
-    } else {
-      this.goToBeginning();
+      const prestartForm = beforeStartForm ? this.prestartForm : null;
+      return this.props.fetchInProgressForm(this.props.formId, this.props.migrations, true, this.props.prefillTransformer, prestartForm);
```

Comment:
What is providing this `fetchInProgressForm` prop? 

If all of the arguments provided to the `fetchInProgressForm` are available in the context where `fetchInProgressForm` given to this component, then I recommend giving this component simple callback instead.

e.g. 

instead of this: 
```
<Component
dataProp={dataProp}
functionProp={functionProp}/>
// and calling this inside of Component like 
this.props.functionProp(dataProp)
```

invert the control: 

```
<Component
callback={() => functionProp(dataProp)}/>
```

This simplifies the component interface and defines the behavior in the context where the data is retrieved. It's not _always_ better to do this- it depends on the context and the story you're trying to tell in it. It's important to group like complexity and behavior.

Reply:
Simplifying this would be nice! Looping @jbalboni for additional insights, while I look into how this would work.

Reply:
Thanks again for your feedback @rianfowler, a ticket has been created (using your comment) to simplify fetchInProgressForm [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10754).

Reply:
@cvalarida good point about restoring the if/else format, updated.